java/query/query-core/src/main/java/sleeper/query/core/tracker/QueryStatusReportListener.java

```diff
@@ -21,25 +21,79 @@
 
 import java.util.List;
 
+/**
+ * Interface for reporting on the status of a query.
```

Comment:
"Interface for" can be omitted, see the Oracle style guide under "omit the subject and simply state the object": https://www.oracle.com/uk/technical-resources/articles/java/javadoc-tool.html

This seems to suggest this is used to create reports, which is not true. The comments on the methods also reflect this, and also need to be updated.

This tracks changes to the status of a query, and is implemented by something that should be notified when the status changes. For example that includes the query tracker, and the web socket for a query that was made through a web socket.